elated-egrets/bangazon-internal-site-2.0
# Added local search

## DESCRIPTION
close #2 
## How to pull down
Commands

```
git fetch --all
git checkout KFLocalSearch
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. Create user account with city in the profile
1. Add 2 items for sale, make sure one is set for local delivery and one is not
1. Create second user account, with a different city listed in the profile
1. Create another item listed for local delivery
1. Type in the name of the city listed in your user profile into the local search bar, press search local products
1. Make sure that only the items from the city you searched for and listed for local delivery display

## What you changed
Added item to navbar template, added logic to list product view

## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Have 2 teammates approved the PR?

Then Ship It!
